packages/fxa-auth-server/config/index.ts

```diff
@@ -137,6 +137,33 @@ const conf = convict({
       format: Boolean,
     },
   },
+  appleAuthConfig: {
+    clientId: {
+      default: 'com.mozilla.firefox.accounts.auth',
+      env: 'APPLE_AUTH_CLIENT_ID',
+      format: String,
+      doc: 'Apple auth client id',
```

Comment:
This reminds me - did we make a ticket somewhere to update our docs on how all of this works together? I don't see one in the epic, it'd be great to have a high level overview of some of this once it's finished 👍